### ISSUE TITLE
PHP 8 Support

### ISSUE THREAD

**Issue by smanhoff** [markitosgv/JWTRefreshTokenBundle#227]
Will this package be updated to support PHP 8?

For example the `doctrine/persistence` package needs to be updated:

ea: replace `Doctrine/Common/Persistence` namespace with `Doctrine/Persistence` in `RefreshTokenManager`.

**Comment by fd6130** [comment]
Is the issue of "Doctrine/Common/Persistence" relevant to #200  ?

**Comment by sanhofman** [comment]
Yes seems relevant and I can confirm the issue is fixed in latest release, thanks 👍

**Comment by fd6130** [comment]
> Yes seems relevant and I can confirm the issue is fixed in latest release, thanks

I think this issue can be close? Cause now already support PHP 8.

**Comment by sanhofman** [comment]
Yes can be closed, thanks!
